.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Is this a bug report or feature request?**
+* Bug Report
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How to Reproduce**
+Steps to reproduce the behavior:
+1. Deploy AuthService ...
+2. Perform this action ...
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Config Files**
+Please provide all relevant configuration. This includes:
+- AuthService configuration.
+- OIDC Provider configuration.
```

Comment:
I assume that you refer to Dex here, but people may have other OIDC providers, so we can't expect to share their configurations. I'd suggest we convert this to "Please provide all the relevant configuration that you can publicly share".